GitPlugin/Connect.cs

```diff
@@ -236,7 +236,7 @@ private void GitPluginUISetup()
                 this.GitPluginUISetupCommandBar();
                 this.GitPluginUISetupContextMenu();
 
-                //    /*
+                // /*
```

Comment:
:bulb: Better to surround this block with `#if false` and remove the `//` from each line